lanl/podman-quadlets/roles/configs/templates/coredhcp/coredhcp.yaml.j2

```diff
@@ -0,0 +1,8 @@
+server4:
+  plugins:
+    - server_id: {{ coredhcp_server_id }}
+    - dns: {{ coredhcp_dns_server }}
+    - router: {{ coredhcp_router }}
+    - netmask: {{ coredhcp_netmask }}
+    - coresmd: https://{{ cluster_name }}.{{ cluster_domain }}:8443 http://{{ cluster_boot_ip }}:8081 /root_ca/root_ca.crt {{ coredhcp_cache_validity }} {{ coredhcp_lease_duration }}
+    - bootloop: /tmp/coredhcp.db {{ coredhcp_tmp_lease_duration }} {{ coredhcp_dhcp_pool }}
```

Comment:
If you want to use CoreDHCP with coresmd v0.2.0, the only change you'll need to make is on this line. You only need to add `default` as the second argument to tell the built-in TFTP server to serve the built-in reboot iPXE script.

Replace:
```yaml
- bootloop: /tmp/coredhcp.db {{ coredhcp_tmp_lease_duration }} {{ coredhcp_dhcp_pool }}
```
with:
```yaml
- bootloop: /tmp/coredhcp.db default {{ coredhcp_tmp_lease_duration }} {{ coredhcp_dhcp_pool }}
```

Reply:
Added a variable `coredhcp_custom_ipxe` that is set to default. Updated the coredhcp template to use it